webpack.dev_container.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = env => {
   const devConfig = devConfigFn(env);
   return _.merge(devConfig, {
     watchOptions: {
-      ignored: /node_modules|bower_components/,
+      ignored: /node_modules|bower_components|\.#|~$/,
```

Comment:
what are the last options u have added?

Reply:
hahaha they are emacs temp files.. without these exception stupid webpack sometimes stop refreshing (watched temp files cause crash) . so i added these options.